causalml/inference/nn/__init__.py

```diff
@@ -1 +1,2 @@
-from .dragonnet import DragonNet
\ No newline at end of file
+from .dragonnet import DragonNet
+from .cevae import CEVAE_MODEL
```

Comment:
This should be changed to `CEVAE` to reflect the name change in `cevae.py`.

Reply:
Thanks for catching it!